ts_app_struct/network_constructor/mix.exs

```diff
@@ -0,0 +1,34 @@
+defmodule NetworkConstructor.Mixfile do
+  use Mix.Project
+
+  def project do
+    [
+      app: :network_constructor,
+      version: "0.1.0",
+      elixir: "~> 1.6.2",
+      start_permanent: Mix.env == :prod,
+      deps: deps()
+    ]
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      extra_applications: [:logger],
+      mod: {NetworkConstructor.Application, []}
+    ]
+  end
+
+  # Run "mix help deps" to learn about dependencies.
+  #Change dependency path accordingly.
+  defp deps do
+    [
+      {:input_parser,  path: "/home/bgargi/Videos/input_parser"},
```

Comment:
again the path has to be relative to the code base and must be valid under all execution environments.